packages/components/src/components/assets/CoinLogo/coins.ts

```diff
@@ -9,6 +9,7 @@ export const COINS = {
     dash: require('../../../images/coins/dash.svg'),
     dgb: require('../../../images/coins/dgb.svg'),
     doge: require('../../../images/coins/doge.svg'),
+    dsol: require('../../../images/coins/sol.svg'),
```

Comment:
Please define separate icon for Solana Devnet, consistent with testnet coins. It can be 
```
<svg viewBox="0 0 40 40" fill="none" xmlns="http://www.w3.org/2000/svg">
    <circle cx="20" cy="20" r="20" fill="#E75F5F" />
    <path fill-rule="evenodd" clip-rule="evenodd"
        d="M13.8123 24.55C13.8741 24.4833 13.9488 24.4298 14.0318 24.3925C14.1149 24.3553 14.2046 24.3352 14.2957 24.3333H29.4831C29.5475 24.3342 29.61 24.3536 29.6635 24.3891C29.717 24.4246 29.759 24.4748 29.7846 24.5335C29.8101 24.5923 29.8181 24.6571 29.8074 24.7202C29.7969 24.7834 29.7682 24.8421 29.7249 24.8894L26.5056 28.45C26.4438 28.5166 26.3692 28.5702 26.2861 28.6074C26.2031 28.6447 26.1133 28.6648 26.0223 28.6666H10.8251C10.7609 28.6657 10.6982 28.6464 10.6448 28.6109C10.5913 28.5753 10.5493 28.5252 10.5237 28.4664C10.4982 28.4077 10.4903 28.3429 10.5008 28.2797C10.5114 28.2166 10.5401 28.1578 10.5834 28.1105L13.8123 24.55ZM29.7394 21.5985C29.7827 21.6458 29.8114 21.7045 29.8219 21.7677C29.8326 21.8308 29.8246 21.8956 29.7991 21.9544C29.7735 22.0132 29.7315 22.0633 29.678 22.0988C29.6245 22.1343 29.562 22.1537 29.4976 22.1546L14.3054 22.1666C14.2143 22.1648 14.1246 22.1447 14.0415 22.1074C13.9585 22.0702 13.8838 22.0166 13.822 21.95L10.5786 18.4015C10.5353 18.3542 10.5066 18.2954 10.496 18.2323C10.4854 18.1691 10.4934 18.1043 10.5189 18.0456C10.5444 17.9868 10.5865 17.9367 10.6399 17.9011C10.6934 17.8656 10.756 17.8463 10.8203 17.8454L26.0126 17.8333C26.1037 17.8352 26.1934 17.8553 26.2765 17.8925C26.3595 17.9298 26.4342 17.9833 26.496 18.05L29.7394 21.5985ZM13.8123 11.55C13.8741 11.4833 13.9488 11.4298 14.0318 11.3925C14.1149 11.3553 14.2046 11.3352 14.2957 11.3333L29.4929 11.3454C29.5571 11.3463 29.6198 11.3656 29.6733 11.4011C29.7267 11.4367 29.7688 11.4868 29.7942 11.5456C29.8197 11.6043 29.8277 11.6691 29.8172 11.7323C29.8065 11.7954 29.7779 11.8542 29.7345 11.9015L26.5056 15.45C26.4438 15.5166 26.3692 15.5702 26.2861 15.6074C26.2031 15.6447 26.1133 15.6648 26.0223 15.6666H10.8251C10.7609 15.6657 10.6982 15.6464 10.6448 15.6109C10.5913 15.5753 10.5493 15.5252 10.5237 15.4664C10.4982 15.4077 10.4903 15.3429 10.5008 15.2797C10.5114 15.2166 10.5401 15.1578 10.5834 15.1105L13.8123 11.55Z"
        fill="white" />
</svg>
```